Mjolnir/src/Subsystems/Chassis.h

```diff
@@ -13,24 +13,28 @@
 #define CHASSIS_H
 #include "Commands/Subsystem.h"
 #include "WPILib.h"
+#include "Commands/TankDrive.h"
+
 
 /**
  *
  *
  * @author ExampleAuthor
  */
-class Chassis: public Subsystem {
+class Chassis: public frc::Subsystem {
 private:
 	// It's desirable that everything possible is private except
 	// for methods that implement subsystem capabilities
 	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-	std::shared_ptr<CANTalon> right1Wheel;
-	std::shared_ptr<CANTalon> right2Wheel;
-	std::shared_ptr<CANTalon> left1Wheel;
-	std::shared_ptr<CANTalon> left2Wheel;
+	static std::shared_ptr<WPI_TalonSRX> left1Wheel;
```

Comment:
Change these declarations as well. This will also cause the definitions to change in the .cpp file. 

   WPI_TalonSRX left1Wheel;

Also, remove `static`. Shouldn't be needed either.